javascript/code-challenges/challenges-02.test.js

```diff
@@ -0,0 +1,343 @@
+'use strict';
+
+/* ------------------------------------------------------------------------------------------------
+
+CHALLENGE 1 - Review
+
+Write a function named raisedToTheThird that takes in an array of numbers and returns a new array of each of those numbers raised to the 3rd power (hint: look up Math.pow()). Use forEach to solve this problem.
+
+------------------------------------------------------------------------------------------------ */
+
+function raisedToTheThird(numbers) {
+  let result = [];
+  numbers.forEach(function(number) {
+    var power = Math.pow(number, 3);
+    result.push(power);
+  });
+
+  return result;
+}
+
+
+
+
+
+
+
+
+
+
+
+/* ------------------------------------------------------------------------------------------------
+CHALLENGE 2
+
+Write a function named addOne that, given an array of numbers, uses map to return a new array with each value simply incremented by 1.
+------------------------------------------------------------------------------------------------ */
+let arr = [1, 2, 3, 4, 5];
+let addOne = (arr) => {
+  let arr1 = [];
+    return arr1;
+
+
+
+let addOneAndrea = (arr) => {
+  return arr.map(n => n+1);
+}
```

Comment:
Good work! Please make sure your work gets added into the addOne function so the tests can run properly!